editor/filesystem_dock.cpp

```diff
@@ -2024,10 +2023,10 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, Vector<Str
 
 	if (p_paths.size() >= 1) {
 		if (!all_favorites) {
-			p_popup->add_item(TTR("Add to favorites"), FILE_ADD_FAVORITE);
+			p_popup->add_item(TTR("Add To Favorites"), FILE_ADD_FAVORITE);
```

Comment:
I don't think we should add an uppercase letter to "to" or "from". This is not common.
For other words it's ok.

Reply:
Every other case of "to" in a popup menu in the editor is capitalized, there's even one instance in the same menu as the above is in: "Move To..."

Reply:
I think that's an error. 

On the same list you also have "Show in File Manager", where the "in" is not capitalized.
You have also:
- Project > "Quit to Project List"
- Debug > "Deploy with ..."
- Debug > "Small Deploy with..."

It seems like the "To" is capitalized only when you have "..." just after. 
Like "Scene > Convert To..." 

But I think that should be changed too. To be see by the documentation team.

Reply:
Fair enough, uncapitalized "to" and "from".